edXRegressionAutomated/Tests/Studio/pages/courseoutlinepage.py

```diff
@@ -0,0 +1,93 @@
+from bok_choy.page_object import PageObject
+from selenium.webdriver.common.keys import Keys
+from ...LMS.pages.courseware import CoursewarePage
+from unitpage import UnitsPage
+
+class CourseOutlinePage(PageObject):
+    """
+    Course Outline page of Auto course
+    """
+    url = None
+
+    def is_browser_on_page(self):
+        return 'course outline' in self.browser.title.lower()
+
+    def click_course_link(self):
+        # Click course link
+
+        self.q(css='.course-link').first.click()
+        CourseOutlinePage(self.browser).wait_for_page()
+
+    def delete_sections(self):
+        # Delete all sections on the page
```

Comment:
Rather than a comment, this should be a docstring. (Use the triple-quote instead of the `#`)